runtime/Java/src/org/antlr/v4/runtime/VocabularyImpl.java

```diff
@@ -178,7 +178,7 @@ public String getSymbolicName(int tokenType) {
 
 	@Override
 	public String getDisplayName(int tokenType) {
-		if (tokenType >= 0 && tokenType < displayNames.length) {
+		if (tokenType >= Token.MIN_USER_TOKEN_TYPE && tokenType < displayNames.length) {
```

Comment:
❗️ This prevents a user from assigning a custom display name to `Token.INVALID_TYPE`. Adding this as a restriction doesn't seem to provide any benefit to the user.